app/src/main/java/com/example/ide/ui/screen/ChatScreen.kt

```
@@ -313,227 +298,227 @@ fun ChatScreen(viewModel: MainViewModel) {
-    
-    for (match in matches) {
-        val language = match.groupValues[1]
-        val code = match.groupValues[2].trim()
-        codeBlocks.add(language to code)
-    }
-    
-    return codeBlocks
-}
+
+@Composable
+fun ChatMessageItem(
```

detekt (1.23.8)

[warning] The function ChatMessageItem appears to be too complex based on Cyclomatic Complexity (complexity: 37). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)